docs/guide/index.md

```diff
@@ -28,7 +28,7 @@ First, create a `package.json` file with `npm init -y` in the root of your theme
 Next, install Vite and the Shopify plugin via NPM.
 
 ```bash
-npm i -D vite vite-plugin-shopify
+npm i -D vite vite-plugin-shopify @by-association-only/vite-plugin-shopify-clean
```

Comment:
@montalvomiguelo I think it would be nice to have such section, however in this instance I think this is a better default behavior as it doesn't change anything for a theme developer but does improve the experience for someone doing theme maintenance.
Either way if you decide to add the section I can help updating the docs :)

Reply:
@hcmlopes Let's move it to a troubleshooting page. Would you be able to help us start that page 🙏

Reply:
Awesome. I will work on updating the PR shortly